worldguard-core/src/main/java/com/sk89q/worldguard/commands/region/RegionCommands.java

```diff
@@ -1111,8 +1111,9 @@ public void teleport(CommandContext args, Actor sender) throws CommandException
             }
         }
 
+        String message = existing.getFlag(Flags.TELE_MESSAGE);
```

Comment:
```
String message = existing.getFlag(Flags.TELE_MESSAGE);
if (message == null || message.isEmpty()) {
  message = Flags.TELE_MESSAGE.getDefault();
}

player.teleport(teleportLocation,  message.replace("%id%", existing.getId()), ...);
```

(For future reference, this is a pretty common pattern, get the thing, check if it's missing, assign it if it is)

Reply:
No, `isEmpty()` means no message.

_The message can be disabled by setting an empty message._

Reply:
I have replaced the ternary operator by an if statement and added commets to make everything clear. Done.